### PR TITLE
prepare v4.3.0 release

### PR DESCRIPTION
**Pull request by mo-rieger**
version file has already been updated in master

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ## unreleased
 
+## v4.3.0 - 2022.08.24
+* Update CSI driver for Kubernetes 1.24
+[[GH-457]](https://github.com/digitalocean/csi-digitalocean/pull/457)
+* Clarify release documentation on branch releases
```
Comment (timoreimann) [inline CHANGELOG.md:6]: That's something we could exclude since that change does not impact the release artifact.

**Review by timoreimann — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mo-rieger) [reply CHANGELOG.md:6]: I assume that you are referring only to l.6?

Comment (khayyamsaleem) [reply CHANGELOG.md:6]: yes @mo-rieger

Comment (timoreimann) [reply CHANGELOG.md:6]: Yes.

**Pull request merged**